packages/oak-addon-richtext-field/lib/core/Editor/html.js

```diff
@@ -41,6 +41,18 @@ const ALIGNMENTS = {
   'text-justify': 'justify',
 };
 
+const addAttrsToChildren = (child, attrs) => {
+  if (child.children) {
```

Comment:
maybe we can "simplify" this function a bit:

```js
const addAttrsToChildren = (child, attrs) => child.children ? {
  ...child,
  children: child.children.map(c => ({
    ...(c.children ? addAttrsToChildren(c, attrs) : {}),
    ...attrs,
  }),
} : child;
```

Reply:
can i do this ? cause the check of `c.children` property are already done in `addAttrsToChildren` func, when she is recall
```diff
- ...(c.children ? addAttrsToChildren(c, attrs) : {}),

+ ...addAttrsToChildren(c, attrs),
```

And if i return an empty Object i have an error
```
Unexpected hyperscript child object: [object Object]
```

Reply:
yep sry the parent condition was added after I wrote the `...(a ? b : c)` one